ExampleTestProject/Assets/Testing/Tests/Shared/TestTitleDataLoader.cs

```diff
@@ -102,7 +102,7 @@ private static void SetTitleId(bool setPlayFabSettings)
 #if ENABLE_PLAYFABSERVER_API || ENABLE_PLAYFABADMIN_API || UNITY_EDITOR || ENABLE_PLAYFAB_SECRETKEY
             PlayFabSettings.DeveloperSecretKey = _loadedData.developerSecretKey;
 #endif
-            PlayFabSettings.ConnectionString = _loadedData.connectionString;
```

Comment:
Will we not need to add the Endpoint equivalent here?

Reply:
This file was added by mistake. Checkout the new template in targets/unity-v2/templates/TestTitleDataLoader.cs